bucket/get_iplayer.json

```diff
@@ -0,0 +1,43 @@
+{
+    "version": "3.25.0",
+    "description": "A utility for downloading TV and radio programmes from BBC iPlayer and BBC Sounds",
+    "homepage": "https://github.com/get-iplayer/get_iplayer",
+    "license": "GPL-3.0-only",
+    "url": "https://github.com/get-iplayer/get_iplayer_win32/releases/download/3.25.0/get_iplayer-3.25.0-setup.exe",
+    "hash": "4c8cabcb3aebc8331152a69ec4a07ccd88dbea9ac2885468e338894fa1aa2f25",
+    "innosetup": true,
+    "bin": [
+        "get_iplayer.cmd",
+        "get_iplayer_pvr.cmd",
+        "get_iplayer_web_pvr.cmd"
+    ],
+    "shortcuts": [
+        [
+            "get_iplayer.cmd",
+            "get_iplayer",
+            "--search dontshowanymatches && cd %HOMEDRIVE%%HOMEPATH% && cmd.exe /k get_iplayer.cmd --help",
+            "get_iplayer.ico"
+        ],

```

Comment:
What is this for?

Reply:
This is to replicate the shortcut that the installer creates at this line:

https://github.com/get-iplayer/get_iplayer_win32/blob/a1fbb3a93206240463b0392e1a0498d7f64ce7ef/get_iplayer.iss#L107